docs/tutorials/notebooks/Monge_Gap.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #92.        bool_plot_pred = "mapped_source" in batch.keys()
</pre><p>No need to call <code>.keys()</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>